otel-infrastructure-collector/k8s-helm/values.yaml

```diff
@@ -13,6 +13,11 @@ opentelemetry-collector:
   fullnameOverride: otel-infrastructure-collector
   clusterRole:
     name: "otel-infrastructure-collector"
+    create: true
+    rules: 
+    - apiGroups: ["", "events.k8s.io"]
+      resources: ["events"]
+      verbs: ["watch", "list"]
```

Comment:
because we enable the `clusterMetrics` preset , it creates a cluster role with watch and list on events resource, for "" apigroup, isnt that enough ? see: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/templates/clusterrole.yaml#L23

Reply:
Users can disable clusterMetrics as we're doing internally :)

Reply:
but if they dont ,  they have 2 clusterroles ?

Reply:
right , and i see the condition is with 'or' , so it wont be duplicated , great!